src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -448,6 +547,33 @@ class adminLogs extends viewModelBase {
     onOpenSettings() {
         this.loadLogsConfig();
     }
+
+    onOpenDownload() {
+        this.startDate(null);
+        this.endDate(null);
```

Comment:
do we need this?

Reply:
yes - this clears the date picker content